themes/default/templates/linkrel;misc;default

```diff
@@ -93,9 +93,15 @@ ELSIF env.script_name.search('journal\.pl');
 
 
 ELSIF env.script_name.search('messages\.pl');
+	IF constants.plugin.Messages;
+		tmp = BLOCK -%]
+<script src="[% constants.real_rootdir %]/messages.js" type="text/javascript"></script>
+		[%- END; linkrels.push(tmp);
+	END;
```

Comment:
Nah, check a line or two above. It only loads if the calling script is messages.pl. Or more precisely if it has the string "messages.pl" in the url somewhere.